src/components/Form/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const Button = ({
+  icon = "",
+  text = "Click me!",
```

Comment:
Maybe we make the default "Submit" here?

Reply:
Yeah, definitely a more usable default than "Click Me!"  🙃